src/web-auth/captcha.js

```diff
@@ -38,14 +45,36 @@ function handleAuth0Provider(element, options, challenge, load) {
   });
 }
 
-function injectRecaptchaScript(element, lang, callback) {
+function globalForRecaptchaProvider(provider) {
+  switch (provider) {
+    case RECAPTCHA_V2_PROVIDER:
+      return window.grecaptcha;
+    case RECAPTCHA_ENTERPRISE_PROVIDER:
+      return window.grecaptcha.enterprise;
+    default:
+      throw new Error('Unknown captcha provider');
```

Comment:
IMO this line doesn't need to be tested, but the linter required I add a default case. Open to testing it I suppose, though I'm not a big fan of testing lines just because they exist.

Reply:
@amaksimuk I agree. However, to satisfy our code coverage tool, you can add lines here which cause coverage to be ignore (I think that's why we're getting Codecov failures).

This applies to anything else like this that you have to add but isn't necessary to test.

Reply:
🤦🏼 @stevehobbsdev thanks for pointing this out!

Reply:
Thanks!